backend/src/services/organizationService.ts

```diff
@@ -243,11 +243,13 @@ export default class OrganizationService extends LoggerBase {
   }
 
   async generateMergeSuggestions(type: OrganizationMergeSuggestionType): Promise<void> {
+    this.options.log.trace(`Generating merge suggestions for: ${this.options.currentTenant.id}`)
```

Comment:
Since you're in a `LoggerBase`, you can just use `this.log.trace()`

Reply:
updated